dir.props

```diff
@@ -263,10 +263,12 @@
     <DotnetHostString>dotnet-host-</DotnetHostString>
     <DotnetHostFxrString>dotnet-hostfxr-</DotnetHostFxrString>
     <DotnetRuntimeString>dotnet-runtime-</DotnetRuntimeString>
+    <DotnetRuntimeMetaPackageString>dotnet-runtime-dep-</DotnetRuntimeMetaPackageString>
```

Comment:
These names should align.  It doesn't make sense for the variable to be named `RuntimeMetaPackage` but the value to be `runtime-dep`.